snippets/auto_gen/hubl_tags.json

```diff
@@ -6,9 +6,9 @@
     },
     "blog_comments": {
         "body": [
-            "{% blog_comments \"my_blog_comments\"\n\tselect_blog=${select_blog}, \n\tlimit=${limit}\n%}"
+            "{% blog_comments \"my_blog_comments\"\n\tselect_blog=${select_blog}, \n\tlimit=${limit}, \n\tskip_css=${skip_css}\n%}"
```

Comment:
This probably should be a separate issue, but I feel like we should be consistent between the styleguide, Design Manager code editor defaults, and this extension when it comes to tabs vs. spaces. Perhaps the extension should also be configurable like the Design Manager is.